apps/discovery-platform/src/funding-service-api/index.spec.ts

```diff
@@ -307,13 +352,13 @@ describe("test funding service api class", function () {
       await db.saveRegisteredNode(dbInstance, node);
 
       const fundingResponse = await fundingServiceApi.requestFunds({
-        amount: 5,
+        amount: BigInt(5),
         node,
       });
 
       const successfulGetRequestStatusBody: getRequestStatusResponse = {
         accessTokenHash: "hash",
-        amount: "5",
+        amount: BigInt("5").toString(),
```

Comment:
I feel like this is just writing "5" but with extra steps

Reply:
if we are trying to express that a big int can come here in form of a string then thats fine, just in some places it is "5" and in others it is BigInt("5").toString().